_posts/2018-04-09-synching-sia.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Syncing Sia in 73 minutes
```

Comment:
What do you think of the title "Overclocking the Sia Blockchain Sync"?

I think of this as similar to overclocking a CPU or GPU in that it's something anyone can do if they know what they're doing, but it introduces too much instability for the casual user.

I'd prefer to focus less on the "73 minutes" metric. It's impressive to a reader who knows that syncing normally takes like 14 hours, but is not so impressive to a reader coming in without that context or to a reader who thinks bootstrapping is a sensible way to sync.

I think the more impressive metric is the speedup percentage. This is a 76% reduction in sync time, right? To me, that's the more exciting metric.

Reply:
I'm not a huge fan of the term "overclocking"; I associate overclocking with hardware tweaks such as increased voltage, cooling, and CPU multipliers.  What do you think about "Improving the Sia Blockchain Sync"?

Reply:
Or "Improving the performance of the Sia Blockchain Sync"

Reply:
Usually, the editor gets to decide the title so I'll let you make the final decision as you're acting as the editor in this case.